### PR TITLE
Missions SetUp, Fetch Data

### PR DESCRIPTION
**Pull request by Bondok6**
## Hi @Leboroz 

I've done with these issues: 

- #20 
- #18 
- #15 
<hr>

### Thanks for taking the time to review my code.

Gif found on CodeReviewGifs.com hosted on Giphy.com

### CODE REVIEW

File: src/components/missions/MissionsList.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { Table } from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import MissionItem from './MissionItem';
+import getMissionsFromApi from '../../redux/missions/missions-action';
+
+const MissionsList = () => {
+  const missionsData = useSelector((state) => state.missions.missions);
```
Comment (Leboroz) [inline src/components/missions/MissionsList.jsx]: I think there are too many missions here. You should change it to make it more readable.

**Review by Leboroz — CHANGES_REQUESTED** [review]
Hello @Bondok6, 👋 

Good job so far. I learned a lot with this code.

**Review by Leboroz — APPROVED** [review]
Approved ✔️
Hi @Bondok6 ,

Congratulations on completing the changes! 👏🏼 🎉

Cheers and good luck in your next milestones! 🍀

congratulation

File: src/components/missions/MissionsList.jsx (hunk 1 above)
Comment (Bondok6) [reply src/components/missions/MissionsList.jsx]: Hi @Leboroz, thanks. 
I agree 100%., I gonna implement it. 👌

**Pull request merged**